packages/web-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -29,7 +32,7 @@ const Dropdown = (props: SpiritDropdownProps) => {
 
```

Comment:
I am expecting here something like
```js
if(breakpoint) {
  depreacationNotice('Prop breakpoint in Dropdown component is deprecated and will be remove in next major version. Please use fullWidthMode instead');
}
```
where the `deprecation notice` is an imported function that will deal with logging those warnings into the console of the browser or the terminal when in the development environment.